relay-dynamic-config/src/defaults.rs

```diff
@@ -143,6 +147,66 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/data.llm.llm_completion_tkens@none".into(),
```

Comment:
Is the missing `o` in `tokens` a typo or the correct metric name?

Reply:
mentioned in description -- `token` is one of our default field regexes that get scrubbed, so the name cannot have the word `token` in it. (Scrubbing happens after metrics extraction, but we want this value to be on the indexed data as well).